raspberry/data/README.md

```diff
@@ -2,20 +2,22 @@
 
 CSV files with the following columns:
 
-- `Time-stamp`
-- `T_ref`
-- `yaw` (rad)
-- `pitch`
-- `roll`
-- `ToF measurement` (mm)
-- `accX`, `accY`, `accZ`, 
-- `Ref alti`
-- `RC data` (string)
-- `mot_FL`
-- `mot_FR`
-- `mot_BL`
-- `mot_BR`
-- `KF alti est`
-- `KF vel est`
-- `KF alpha est`
-- `KF beta est`
+- `timestamp_ms` timestamp in milliseconds
+- `throttle_ref` throttle reference from the RC
+- `yaw` yaw angle in rad
+- `pitch` pitch angle in rad
+- `roll` roll angle in rad
+- `tof_measurement` measurement from the ToF sensor in mm
+- `accX` acc_x from the accelerometer, in m/s^2
+- `accY` acc_y from the accelerometer, in m/s^2 
+- `accZ` acc_z from the accelerometer, in m/s^2
+- `altitude_ref` altitude reference in m
+- `RC data` dump of the RC data as a string
```

Comment:
This is the encoded data from the parser. So, can you please mention that in line 15? Otherwise, we might forget this later.